eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -129,8 +129,9 @@ steps:
 
   - script: |
       chmod -R a+r .
-      ../dotnet publish -bl:MauiAndroid.binlog -f net6.0-android -c Release
-      mv ./bin/Release/net6.0-android/com.companyname.MauiTesting-Signed.apk ./MauiAndroidDefault.apk
+      ../dotnet restore
+      ../dotnet publish -bl:MauiAndroid.binlog -f net6.0-android -c Release -r android-arm64 --no-restore --self-contained
```

Comment:
`--self-contained` shouldn't be needed. https://github.com/dotnet/sdk/issues/23705 is now closed.

Reply:
Cool, I will try it without it.

Reply:
I think this pipeline is using the .NET 6.0.200 SDK, it looks like it is fixed in 6.0.300.

Reply:
We should be using the latest SDK.... whatever our customers will be using.

Reply:
Yes, I'll let Parker know when it's time to switch.

We're currently blocked on some missing dotnet/linker changes: https://github.com/xamarin/xamarin-android/pull/6742